### ISSUE TITLE
MAPPACK: Create a community map pack compatible with WoP 1.7

### ISSUE THREAD

**Issue by kai-li-wop** [PadWorld-Entertainment/worldofpadman#340]
Custom maps list for World of PADMAN. All we need is a suitable license to make necessary changes like CC-BY-NC or GPL 3.0.

### To be implemented with upcoming version 1.2.0
- [ ] Marble Mace by Iltis (GPL-3.0)
- [ ] Quadwell by Victor Karp (CC-BY-NC-SA)

### Implemented with version 1.1.0
- [x] Cheese Dome by Iltis (GPL-3.0)
- [x] Copy and Paste by 3aTmE! (CC-0)
- [x] Dark Place by Iltis  (GPL-3.0)
- [x] Fly or Die by BuddyFriendGuy (GPL-3.0)

### Implemented with version 1.0.0
- [x] 7 Colors by Hectic (GPL-3.0)
- [x] A Long Way by Hectic (GPL-3.0)
- [x] Coloor by 3aTmE! (CC-BY-NC)
- [x] Cube Fight by Hectic (GPL-3.0)
- [x] FragBox by 3aTmE! (CC-BY-NC)
- [x] Hurry up! by Hectic (GPL-3.0)
- [x] Pac-Man by Hectic (GPL-3.0)
- [x] Tiny CTL by blu (WTFPL)
- [x] Workshop by Hectic (GPL-3.0)

### Could be implemented later
- [ ] Frag A Pad by Iltis (GPL-3.0)
- [ ] Freerun2 by Sauerbraten & Iltis (GPL-3.0)
- [ ] Elevator by Hectic (GPL-3.0)
- [ ] Jum'n'Run by Hectic (GPL-3.0)
- [ ] Backyard Kerfuffle by 3aTmE! (CC-BY-NC) --> map file is missing
- [ ] Dead Simple by 3aTmE! (CC-BY-NC) --> map file is missing
- [ ] Oasis by Wakey (GPL-3.0)
- [ ] 1 vs 1 DachArena by Nasbit (CC-BY-NC) --> map file is missing
- [ ] DeadHouse by Nasbit (CC-BY-NC) --> map file is missing
- [ ] StrokeFarm grey version by Nasbit (CC-BY-NC) --> map file is missing
- [ ] StrokeFarm light white version by Nasbit (CC-BY-NC) --> map file is missing
- [ ] Underground Arena by Nasbit (CC-BY-NC) --> map file is missing
- [ ] Underground Arena anti-color version by Nasbit (CC-BY-NC) --> map file is missing

### License unclear or proprietary:
- garage by Schakal (?)
- aster kitchen by nestel (?)
- p95 by nestel (?)
- holidays by nestel (?)
- padelectric by nestel (?)
- padanita by nestel (?)
- the padman2099 by nestel (?)
- the bowling by nestel (?)
- jam 23 by nestel (?)
- the tower by nestel (?)
- paint arena by nestel (?)
- dana's room by nestel (?)
- ludo city by nestel (?)
- 1vs1_oregano by oregano4 (?)
- 1vs1 DM17 by GLL (?) --> map file available
- 2 Türme by GLL (?)
- monkroom3c2 by Padbabs (c)
- sahasplace by Saha (?)
- pokernight by Phobos (?)
- shopping center by Phobos (?)
- award theater by Phobos (?)
- mushroom garden by Liorlevi99 (?)
- toxic fabric by DevilsRightHand (?)
- todestonne by Blu (?) --> map file available
- aquaria land by PaDbUrY (?)
- casino by PaDbUrY (?)
- Battlezone by Pudding (?)
- Block City by Wiesel (?)
- Blockzone by DarthNihilus (?)
- Cave by Blu (?)
- Dächer by Bowler (?)
- Kegelbahn by Bowler (?)
- castle war by Jables007 (?)
- classroom by method (?)
- color room by Sir_Bart95 (?)
- Deck 7 by Toby (?)
- Training by Hectic (?)
- hobby room by the wiesel (?)
- WTF by Horny (?)
- Hundemap (???)
- Hyrule Castle by Blu (?)
- jump and run by Bury (?)
- PadOffice Killroom (???)
- Kontestmap by Gubba (?)
- Kid's Room by Method (?)
- Laboratory by The Wiesel (?)
- Box Yard by LK (?)
- DM17 by LK (?)
- Banana by Butterschmalztorte (?)
- Bob by Butterschmalztorte (?)
- Marble Hall by Hirnschinken (?)
- The old Home (???) --> map file available
- oldcastle by Toby (?)
- padarena by ChrisEU (?)
- Doenerhut! by Phobos (?)
- DM17 by Phobos (?)
- DM17 remake in color by Phobos (?)
- Phobos room by Phobos (?)
- Plattformen by Silent (?)
- Portal Stage DM17 by Bury (?)
- Private Property PaDbUrY's (?)
- Puds DM17 by Pud (?)
- Race Combat by Pud (?)
- Rainbow 2 by Nils (?)
- Skatefest by The Wiesel (?)
- Strafes by Lucky & Fokili (?)
- The Arena by Bury (?)
- The Match (???) --> map file available
- The Wod (???) --> map file available
- Eyecancer map by Toby (?)
- Toby's Room by Toby (?)
- Tournament by Horny (?)
- Underwatter by MapperSkyGuy (?) --> map file available
- Walrussel's_DM17 by Walrussel (?)
- Wentolos Room by Wentolo (?)
- Wentolos Rooms by Wentolo (?)
- aircombat by magic (?)
- cyberroom (???)
- fun map by Schneeball (?)
- geb2 by Tiko (?)
- PadCellar by OSMO (?)
- Playground by Gametot (?)
- Sandfestung by MachoPig (?)
- Something by vIiRuS (?)
- Kino Map by Spider (?)
- Staub2 by Schmitt (?)
- test coridor (???)

**Comment by tuxaddict** [comment]
again, another great idea! i think you could delete "Flat by Saha (?)" because as far as i can remember, i did only finish the  "sahasplace by Saha (?)" and both titles seem to refer to the same map:
![usax](https://github.com/user-attachments/assets/48fd5f88-f216-40c8-8165-7cfc70a2b0ca)

i may have a look into it, since i stumbled over https://www.nvidia.com/de-de/geforce/rtx-remix/ and this may be a great playground... thank you @kai-li-wop

**Comment by kai-li-wop** [comment]
Done for now.

**Comment by cinnamon189** [comment]
Is this the right place to make new suggestions? If so, I would like to suggest: TILES, MUSHROOMGARDEN - RC, STCHDM1 (Poolshark), SHKIPER3DM1 (Bathhouse) and plutonians.

**Comment by kai-li-wop** [comment]
Yes, you can. Only maps with the map file available and with proper license like gpl, cc etc. are suitable for the map pack.

**Comment by cinnamon189** [comment]
@kai-li-wop  Okay, thanks. So, if we find a map on an established server, it is safe to assume that the map file is "available", right? If so, how can we find out about its license?

**Comment by mgerhardy** [comment]
> [@kai-li-wop](https://github.com/kai-li-wop) Okay, thanks. So, if we find a map on an established server, it is safe to assume that the map file is "available", right? If so, how can we find out about its license?

No, unfortunately it doesn't work like this. You have to contact the author and asks them to release the radiant map file - and ask for permission to release those map files under the terms of the CC-By-0 or GPL (as mentioned above). The files on the public servers are usually only the final compiled result of the map. This is the pk3 files that "only" contain the bsp files and the companion files (like aas). If you are going to contact the author, please check out the existing readmes in the community map pack to establish some kind of standards of what should go into those readmes. Like e.g. how was the map compiled (which parameters - which versions of the mapper tools and so on)

**Comment by kai-li-wop** [comment]
MapPack v1.1.0 done for now
